### ISSUE TITLE
can we identity and return object from collection base on expression using rules engine

### ISSUE THREAD

**Issue by bala3011s** [microsoft/RulesEngine#35]
```csharp
public class Organization {
   public List<Employee> Employees {get; set; }
}

public class Employee{
   public int Age {get; set; }
}
```

Passing object of Organization to rule engine. i have to take employee from list base on age

**Comment by abbasc52** [comment]
Currently we don't support this. We are planning to achieve it via #23 .

Are you planning to use only expression evaluation? or you want to execute some rule and return custom output?

**Comment by bala3011s** [comment]
Want expression evaluation and as well ability to hook some rule or custom code like ReSettings.
Also want output of one rule to as input for another rule, so we can build solution like workflow with activity or steps.

**Comment by abbasc52** [comment]
This functionality will be available as a part of #23 
I am closing this thread, if you need any specific functionality, you can add it in the above mentioned issue
